### PR TITLE
Cloud 3697 open nebula vmcount

### PR DESCRIPTION
**Pull request by ccamp2-7**

### CODE REVIEW

File: actions/cluster_vm_count.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                    vm_count += len(host.VMS.ID)
+
+            else:
+                cluster_value = [clst.HOSTS.ID]
+                host = one.host.info(cluster_value)
+                vm_count = len(host.VMS.ID)
+
+            result[clst.NAME] = vm_count
+
+        return result
```
Comment (jschoewe) [inline actions/cluster_vm_count.py:44]: Some of these files need a blank line at the end or they'll fail the flake8 tests when you run make

File: actions/cluster_vm_count.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+description: Retrieves the hosts on an Open Nebula system.
+enabled: true
+entry_point: cluster_vm_count.py
+parameters:
+    open_nebula:
+        type: string
+        description: >
+            Pre-Configured Open Nebula connection details
+        required: false
+        default: ~
```
Comment (jschoewe) [inline actions/cluster_vm_count.yaml:13]: same here

File: tests/test_action_cluster_vm_count.py [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+        # Mock one object and run action
+        mock_info = mock.Mock()
+        mock_info.CLUSTER = test_clusters
+        mock_one = mock.Mock()
+        mock_one.clusterpool.info.return_value = mock_info
+        mock_session.return_value = mock_one
+        action.run(open_nebula)
+        mock_session.assert_called_with(open_nebula)
+        mock_one.clusterpool.info.assert_called_with()
```
Comment (jschoewe) [inline tests/test_action_cluster_vm_count.py:126]: and here

**Comment by jschoewe** [comment]
LGTM

File: tests/test_action_template_delete.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
```
Comment (bishopbm1) [inline tests/test_action_template_delete.py:1]: Is there a reason we are testing a file that isn't in this PR?

File: tests/test_action_template_get_by_name.py [hunk 5]
```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
```
Comment (bishopbm1) [inline tests/test_action_template_get_by_name.py:1]: Is there a reason we are testing a file that isn't in this PR?

**Review by bishopbm1 — APPROVED** [review]
Code itself looks good just had a few questions

**Review by bishopbm1 — APPROVED** [review]
LGTM

**Pull request merged**